.travis.yml

```diff
@@ -6,11 +6,15 @@ python:
 - "3.7"
 services: docker
 before_install:
+- git rebase HEAD $TRAVIS_BRANCH
 - sudo apt-get update
 - wget http://neurodeb.pirsquared.org/pool/main/s/singularity-container/singularity-container_2.6.1-2~nd16.04+1_amd64.deb
 - sudo apt-get -f install ./singularity-container_2.6.1-2~nd16.04+1_amd64.deb
 - pip install git+https://github.com/virtuald/pyhcl.git@0.3.12#egg=pyhcl
 install:
 - pip install cli/
 script:
-- popper run --recursive
+- export POPPER_TEST_MODE=1
```

Comment:
can we rename the values assigned to this, e.g. `gitless` (or `without-git-repository`) instead of 1